helm/idol-mediaserver/values.schema.json

```diff
@@ -131,6 +135,9 @@
                 "host": {
                     "type": "string"
                 },
+                "path": {
```

Comment:
need servicePath here too.
Could possibly rewrite this entire schema to be based on idol-library-example-test